Week 4 Assignment Relaxr Blog/app.js

```diff
@@ -0,0 +1,47 @@
+$( document ).ready(function() {
+    console.log( "ready!" );
```

Comment:
The ```$.ready``` function should cover all the functions below, since this function is run when your page has loaded all of your DOM, (do you remember what DOM is?).

Hence it should be 

```
$( document ).ready(function() {
    //All of your code, example: 

    $('.button-navy').click(function(event) {
 	 event.preventDefault();
    	 alert("We are not ready for sign-ups yet");
    });
});
```

This can be useful, especially if your page hasn't loaded the ```.button-navy``` elements, it might not correctly add the listeners.

example: http://www.w3schools.com/jquery/event_ready.asp